cdp_client/cdp.py

```diff
@@ -48,8 +49,9 @@ class UnknownError(Exception):
 
 
 class Client:
-    def __init__(self, host, port=7689, auto_reconnect=True):
-        self._connection = Connection(host, port, auto_reconnect)
+    def __init__(self, host, port=7689, auto_reconnect=True, use_encryption=False, encryption_parameters=dict(),
```

Comment:
probably host should also have default '127.0.0.1'

Reply:
Added the default 127.0.01